style.css

```diff
@@ -290,7 +290,16 @@ body {
   color: white;
 }
 
+button:enabled {
+  border: #5f55f7;
+}
+
 button:hover {
   border: #5f55f7;
-  font-weight: bold;
+}
+
+button:disabled {
+  border: #5f55f7;
+  background-color: #2a2b33;
+  color: #c8c8d4;
 }
```

Comment:
- [ ] I think it's better to use  `<section>` tag to wrap the work section instead of a div. And inside it, you can use divs instead of sections. It will make your webpage more semantically correct
![image](https://user-images.githubusercontent.com/43698511/130147261-1aef23dc-0f9d-4901-823e-223ca765efe2.png)